lib/compilers/pony.ts

```diff
@@ -0,0 +1,78 @@
+import path from 'path';
```

Comment:
Argh, the linter check for this is still broken so we need to check manually:

Can you add a license header here please? :)